### PR TITLE
Set up Docker

### PR DESCRIPTION
**Pull request by felixclack**
We want to make it easier for developers to get the app up and running
in their dev environment.

Docker gives us a way to programmatically create an isolated environment
for developing.

Run `bin/docker` to get the app running.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +51,32 @@ Run the application on `http://localhost:3000`:
 bin/dev
 ```
 
+#### Docker
+
+If you prefer to run the application in a Docker container, then you can use the following commands:
+
+```bash
+bin/docker
```
Comment (tvararu) [inline README.md:59]: I love that we're using `bin/` scripts for everything!

File: yarn.lock [hunk 2]
```diff
@@ -17,24 +17,6 @@
...
-  integrity sha512-ZXpxUjCfkdbuXfoGrsFK80qsVzACs8xCfie9rt2jMTSN6o1olXVA0Nrk8u02yNEwSiVJm/4QSOa8cUcMj6VQjg==
-  dependencies:
-    "@hotwired/turbo" "^7.1.0"
-    "@rails/actioncable" "^7.0"
-
-"@hotwired/turbo@^7.1.0":
-  version "7.1.0"
-  resolved "https://registry.npmjs.org/@hotwired/turbo/-/turbo-7.1.0.tgz"
-  integrity sha512-Q8kGjqwPqER+CtpQudbH+3Zgs2X4zb6pBAlr6NsKTXadg45pAOvxI9i4QpuHbwSzR2+x87HUm+rot9F/Pe8rxA==
-
```
Comment (tvararu) [inline yarn.lock:37]: Why are there changes to the lockfile in this PR? Does running yarn inside docker produce different results compared to natively?

**Review by tvararu — APPROVED** [review]

File: bin/docker [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+docker compose up css js web
```
Comment (tvararu) [inline bin/docker:3]: Could you configure your editor to append a newline to the end of files? Most editors do this by default as it's a UNIX convention.

File: yarn.lock (hunk 2 above)
Comment (felixclack) [reply yarn.lock:37]: The lock file was out-of-date. I noticed these changes while setting up the app in Docker.

**Pull request merged**